iml-agent/src/http_comms/crypto_client.rs

```diff
@@ -25,7 +25,7 @@ pub fn get_id(pfx: &[u8]) -> Result<Identity, ImlAgentError> {
 /// * `id` - The client identity to use
 pub fn create_client(id: Identity) -> Result<Client, ImlAgentError> {
     Client::builder()
-        .danger_accept_invalid_certs(true)
```

Comment:
Can you split this change out to it's own patch?

Reply:
Done
https://github.com/whamcloud/integrated-manager-for-lustre/pull/2067